// errors are not skipped by the compiler.
error
func test() {
	print('hi')
}
test()
